specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/registries.json

```diff
@@ -421,6 +467,38 @@
       },
       "additionalProperties": false
     },
+    "EndpointServiceConnectionStatus": {
+      "enum": [
+        "Approved",
+        "Pending",
+        "Rejected",
+        "Disconnected"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "EndpointServiceConnectionStatus",
+        "modelAsString": true
+      },
+      "additionalProperties": false
+    },
+    "PartialManagedServiceIdentity": {
```

Comment:
>PartialManagedServiceIdentity [](http://example.com/codeflow?start=5&length=29)

Refactor suggestion
Would you be able to find an equivalent typedef in 

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json)

to $ref instead?

Reply:
our PartialManagedServiceIdentity is defined differently so we can't use this: [PartialManagedServiceIdentity.cs](https://msdata.visualstudio.com/Vienna/_git/vienna?path=/src/azureml-api/src/RegistryManagement/Contracts/ArmCommon/PartialResources/PartialManagedServiceIdentity.cs&_a=contents&version=GBmaster)